src/components/MainView.js

```diff
@@ -87,12 +97,42 @@ function MainView({ children }) {
           `}
         >
           <ScrollView>
-            <Layout
-              parentWidth={vw - (compactMode ? 0 : MENU_PANEL_WIDTH)}
-              paddingBottom={0}
+            <div
+              css={`
+                display: flex;
+                flex-direction: column;
+                min-height: 100%;
```

Comment:
Should this be `height`? I think generally we use `min-height` when setting a determined px height

Reply:
Both seem reasonable and it may come down to stylistic preference. Perhaps `height` has the advantage here as `min-height` _suggests_ the height may be dynamically adjusted (which is not really true in this case).